FWCore/ServiceRegistry/interface/PathsAndConsumesOfModulesBase.h

```diff
@@ -66,17 +75,38 @@ namespace edm {
       return doModulesWhoseProductsAreConsumedBy(moduleID, branchType);
     }
 
-    // This returns the declared consumes information for a module.
-    // Note the other functions above return a reference to an object
-    // that is held in memory throughout the job, while the following
-    // function returns a newly created object each time.  We do not
-    // expect this to be called during a normal production job where
+    std::vector<eventsetup::ComponentDescription const*> const& esModulesWhoseProductsAreConsumedBy(
```

Comment:
Could you add a comment describing the return value in similar way to the `modulesWhoseProductsAreConsumedBy()` above?